CHANGELOG.md

```diff
@@ -21,6 +21,11 @@ Things to be included in the next release go here.
 ### Added
 
 - Added curve query support for the MSO2KB series scopes
+- Full Python API support for TekScopePC device.
+
+### Fixed
+
+- Fixed APIs with queries syntax missing.
```

Comment:
Add some more details, like the syntax of the query and the model. This needs to explain what changed. I don't know what changed from reading this.

Reply:
Since there was no API support for TekScopePC, we cannot consider these changes as fix.
Remove this as it covered under addition of full support of APIs.